packages/osc-recruitment/app/routes/jobs/index.tsx

```diff
@@ -67,8 +125,32 @@ export default function Index() {
                     <PagePreview query={query} params={params} Component={PageContent} />
                 </PreviewSuspense>
             ) : ( */}
+
             <PageContent {...jobs} />
             {/* )} */}
+
+            <Form
+                className="o-container o-row--2xl o-flex o-flex--center"
+                onChange={handleOptionSelect}
+            >
+                <Select
+                    className={` u-pt-xs c-select--width o-flex--center`}
```

Comment:
Should probably try and find an eslint rule for this 😅 But if you're not interpolating any values can you try and avoid setting props like this. Ideally it should just be a string
```suggestion
                    className="u-pt-xs c-select--width o-flex--center"
```